reference/blob.v1.yaml

```diff
@@ -49,6 +49,57 @@ tags:
       If the blob is created with content or has been updated to include content then its status is `available`.
 
 paths:
+  '/v1/users/{userId}/device-logs':
+    parameters:
+      - $ref: ./common/parameters/tidepooluserid.yaml
+    post:
+      operationId: UploadDeviceLogs
+      summary: Upload new device logs as json
+      description: Uploads a new device logs json to the user's account. The `Digest` header must represent the MD5 hash of the json data content.
+      parameters:
+        - $ref: ./common/parameters/digestmd5.v1.yaml
+        - schema:
+            type: string
```

Comment:
This and `X-Logs-Start-At-Time` should be able to leverage `reference/common/models/datetime.v1.yaml` - see `reference/common/headers/tidepoolsessiontoken.v1.yaml` for example